code/game/jobs/job/medical.dm

```diff
@@ -114,7 +114,11 @@
 	uniform = /obj/item/clothing/under/rank/medical/blue
 	suit = /obj/item/clothing/suit/storage/toggle/labcoat/surgeon
 	shoes = /obj/item/clothing/shoes/surgeon
-	head = /obj/item/clothing/head/surgery/blue
+
+/datum/outfit/job/doctor/surgeon/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	if (isskrell(H) == FALSE)
+		H.equip_to_slot_or_del(new /obj/item/clothing/head/surgery/blue(H), slot_head)
```

Comment:
This isn't an acceptable solution I don't think. It's too hard-coded for my liking. I'd just remove the cap in general and place them in ORs.

Reply:
That's how tajara and unathi procs work as well. It's a bandaid solution until actual Skrell headgear mechanics are implemented.

Reply:
I hate shitcode.